### PR TITLE
added connection to nuts node

### PR DESCRIPTION
**Pull request by woutslakhorst**
Uses nuts-node as client lib!

### CODE REVIEW

File: api/api.go [hunk 1]
```diff
@@ -128,3 +131,28 @@ func (w Wrapper) UpdateServiceProvider(ctx echo.Context) error {
...
+		ServerAddress: w.NodeAddress,
+		Timeout:       1 * time.Second,
+	}
+
+	didDocument, err := nodeClient.Create()
+	if err != nil {
+		return err
+	}
+
+	did := didDocument.ID.String()
```
Comment (reinkrul) [inline api/api.go:150]: when we store customers, do we allow 'connecting' them more than once?

File: api/api.yaml [hunk 2]
```diff
@@ -34,6 +34,26 @@ paths:
...
+      operationId: connectCustomer
+      description: Generate a DID for the customer and store it.
+      responses:
+        200:
+          description: All the information of the current customer.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Customer"
   /web/service-provider:
```
Comment (reinkrul) [inline api/api.yaml:57]: what are customers v.s. service providers?

**Review by reinkrul — CHANGES_REQUESTED** [review]

File: api/api.go (hunk 1 above)
Comment (woutslakhorst) [reply api/api.go:150]: not via UI, but via backend, yes. connect is just generating a new DID

File: api/api.yaml (hunk 2 above)
Comment (woutslakhorst) [reply api/api.yaml:57]: service_provider == vendor

**Pull request closed without merging**